component/exoscale-plans/grafana.json

```diff
@@ -0,0 +1 @@
+{"hobbyist-2":{"note":"grafana hobbyist-2","size":{"cpu":"2","disk":"0","memory":"2Gi"}},"startup-4":{"note":"grafana startup-4","size":{"cpu":"2","disk":"0","memory":"4Gi"}},"startup-8":{"note":"grafana startup-8","size":{"cpu":"4","disk":"0","memory":"8Gi"}}}
```

Comment:
Where is this being used?

Reply:
Currently not at all, I guess they have a new grafana service that gets returned with the api call that returns all version. I decided to keep it for now.